packages/expo-app-auth/android/src/main/java/expo/modules/appauth/AppAuthConstants.java

```diff
@@ -0,0 +1,34 @@
+package expo.modules.appauth;
+
+public class AppAuthConstants {
+  static String ManifestURL = "experienceUrl";
+
+  public interface Error {
```

Comment:
The interface is probably fine since no one implements it, but typically if you want a namespace for constants, a final class is the way to go:

```java
public final class Errors {
  private Errors() {}
  public static final string DEFAULT = "ERR_APP_AUTH";
  ...
}
```